osmosis/assetlist.json

```diff
@@ -16538,29 +16538,218 @@
       ]
     },
     {
-      "description": "Beer Is Good for You!",
```

Comment:
This looks ugly, but the BWH asset is listed twice in master. If you check individual commits, it looks cleaner :)